### DISCUSSION TITLE
Copilot auto-completes only one line when multiple cursors exist in vscode

### DISCUSSION THREAD

**Discussion by reillymck** [community/community#55594] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I'm not sure where the proper place to request this is, but when using the vscode copilot extenstion, I encounter a problem when using multiple cursors. VSCode allows for a user to have multiple cursors, enabling typing on multiple lines simultaneously. Copilot still makes suggestions when I have multiple cursors - which I want. However, when I accept the suggestion, it only applies to the top cursor, instead of all places where I have a cursor. I have attached a video for reference. The video includes the creation of multiple cursors, copilot making a suggestion, and the suggestion being accepted, but only placed on the top line. Please make the suggestion apply for all cursor locations, thus keeping the productivity increase that copilot is meant to have. 
[Screencast from 05-15-2023 08:02:02 PM.webm](https://github.com/community/community/assets/50156344/4e8ddce0-e665-4d6e-aa2b-8af340dd3fdb)

VSCode Version: April 2023 (version 1.78.2)
Copilot Version: v1.86.82
OS: Ubuntu 22.04.2 LTS

**Comment by Akash1134** [comment]
Hi @reillymck ,

Can you please share with the log files so that we can understand your issue better? You can access them here: View > Command Palette... > Github Copilot: Collect Diagnostics as described in this article:
​
[Viewing and collecting log files](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-visual-studio-code#viewing-and-collecting-log-files)

Thanks!

**Comment by nanabear0** [comment]
Having the same issue on Vscode insiders on a Windows 11.

https://github.com/community/community/assets/5321587/42e784bd-c901-4cdc-a7d4-5169540881b1

## Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Self Signed Certificates: disabled
- Send Restricted Telemetry: enabled

## Extensions

- Copilot Version: 1.88.142
- Is `win-ca` installed?: true
- Is `mac-ca` installed?: true

## Node setup

- Number of root certificates: 131
- Operating system: Windows_NT
- Operating system version: 10.0.22621
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 -

**Comment by reillymck** [comment]
As of the newest copilot update, this is now resolved for me. Thanks!

**Reply by RamonBalthazar** [reply]
What version are you on?
I'm still seeing the issue.

**Reply by reillymck** [reply]
Actually I updated vscode and now I have this issue again. Copilot will only predict/autofill prediction for wherever the first cursor is placed. It was working for me previously. 
VS Code version 1.79.2
Copilot version 1.95.233
Ubuntu 22.04
I downloaded vscode through snap.

**Comment by reillymck** [comment]
With newest versions, it is no longer working for me. Here is the results of collect diagnostics:
## Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Self Signed Certificates: disabled
- Send Restricted Telemetry: disabled

## Extensions

- Copilot Version: 1.95.233
- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

## Node setup

- Number of root certificates: 131
- Operating system: Linux
- Operating system version: 5.19.0-46-generic
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 -

**Comment by Kontrolix** [comment]
This issue is still around. Is this planned to be solved?

Here is my Diag:

## Copilot

- Version: 1.144.0
- Build: prod
- Editor: vscode/1.85.1

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Custom Certificates: enabled
- Send Restricted Telemetry: disabled
- Chat: enabled

## Node setup

- Number of root certificates: 140
- Operating system: Darwin
- Operating system version: 23.0.0
- Operating system architecture: arm64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: n/a
- Proxy port: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- api.githubcopilot.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 

## VS Code Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false
